packages/ember-glimmer/tests/integration/content-test.js

```diff
@@ -59,6 +59,102 @@ moduleFor('Static content tests', class extends RenderingTest {
 
 });
 
+moduleFor('Null object tests', class extends RenderingTest {
+  ['@test it should read from an Object.create(null)']() {
```

Comment:
This could probably go into the shared `DynamicContentTest` to test against all the possible positions. Something like:

```javascript
  ['@test it can read from a null object']() {
    let nullObject = Object.create(null);
    nullObject['message'] = 'hello';

    this.renderPath('nullObject.message', { nullObject });

    this.assertText('hello');

    this.assertStableRerender();

    this.runTask(() => set(nullObject, 'message', 'goodbye'));

    this.assertText('goodbye');
    this.assertInvariants();

    nullObject = Object.create(null);
    nullObject['message'] = 'hello';

    this.runTask(() => set(this.context, 'nullObject', nullObject));

    this.assertText('hello');
    this.assertInvariants();
  }

```